src/OSPSuite.Core/Domain/Services/ISimModelBatch.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace OSPSuite.Core.Domain.Services
 {
-   public interface ISimModelBatch : IDisposable
+   public interface ISimModelBatch : IDisposable, ICloneable
```

Comment:
Let's remove this clone stuff please

Reply:
Yes sure, but the current code does not support running the SimulationBatch still. As soon as I complete the code for running also the SimulationBatch I will remove the clone and all the old support.

Reply:
I can't merge the code until this is removed. So please remove, then we can merge, and then we add the other use case. Thanks

Reply:
Done